WooCommerce/WooCommerceTests/Tools/MockOrders.swift

```diff
@@ -118,7 +128,7 @@ final class MockOrders {
                      coupons: [],
                      refunds: [],
                      fees: [],
-                     taxes: [] // TODO: 5809 - Add sampleOrderTaxLines method
+                     taxes: [tax]
```

Comment:
I don't think calculating tax here is necessary either. How about we keep it empty for now and only update it when there are tests that require it?

Reply:
Makes sense. 
Done in https://github.com/woocommerce/woocommerce-ios/pull/5931/commits/561c6f19f9044da4fb8f91796917cf455adab354